cmd/porter/bundle.go

```diff
@@ -92,11 +92,11 @@ The lint command is run automatically when you build a bundle. The command is av
 func buildBundleInstallCommand(p *porter.Porter) *cobra.Command {
 	opts := porter.InstallOptions{}
 	cmd := &cobra.Command{
-		Use:   "install [INSTANCE]",
-		Short: "Install a new instance of a bundle",
-		Long: `Install a new instance of a bundle.
+		Use:   "install [NAME]",
```

Comment:
Would you change this one back to `[INSTALLATION]`? I made a mistake when I told you NAME on our chat today.